src/ocean/time/StopWatch.d

```diff
@@ -99,7 +99,7 @@ public struct StopWatch
 
         ulong microsec ()
         {
-                 return (timer - started);
+                 return (timer >= started)? (timer - started) : 0;
```

Comment:
Yes, I think you're right. I can't find any authoritative source on _integer underflow_ (as oposed to the floating point one) :+1: